web-console/knikubevirt/Create-vm/2.networking/create-vm-networking.md

```diff
@@ -0,0 +1,20 @@
+# Create A Virtual Machine
+
+## Step 2: Networking
+
+![Default](img/3-00.png)
+The user enters the Networking step with an existing NIC pre-defined, set to Type (local). In this step, the 'Review and Create' button is still available to skip the rest of the steps. At this point, the user can select to customize this NIC, add new ones, or remove all.
+
+![Adding NIC](img/3-01.png)
+Click on 'Add NIC' will pop up a "Create NIC" modal. The NIC name is prepopulated with a numeric count from of existing NICs on this flow. The user may change the NIC name. All the fields are preselected except Type. In order for the user to apply and add this NIC, the user must select one of the type options.
```

Comment:
I asked Dan Kenigsberg and he said that We CANNOT use the "default" network for more than one interface. He did say that we can choose one of the other networks (the first one alphabetically?).
For secondary networks, the default type maybe "bridge" and so on.
I don't think it will be helpful since it does not answer the user's need since there is no consistent generic default that is "Better" then the rest of the options.